src/net.cpp

```diff
@@ -555,11 +555,13 @@ void CNode::SweepBanned()
     banmap_t::iterator it = setBanned.begin();
     while(it != setBanned.end())
     {
+        CSubNet subNet = (*it).first;
```

Comment:
Not used a single time currently in net.cpp with (*it), so I'll leave that. But I'm open for further comments on this.